vendor/k8s.io/kubernetes/cmd/libs/go2idl/client-gen/main.go

```diff
@@ -182,7 +207,16 @@ func main() {
 			CmdArgs:             cmdArgs,
 		}
 	} else {
-		inputPath, groups, gvToPath, err := parseInputVersions()
+		var inputPath []string
+		var groups []types.GroupVersions
+		var gvToPath map[types.GroupVersion]string
+		var err error
+
+		if len(*inputVersions) == 1 && len(*groupFlag) > 0 {
```

Comment:
Dous this limit us to one clientset per apigroup and version? Would be nice to keep the flexibility for real clientsets.

In fact, I would prefer to just patch https://github.com/openshift/origin/pull/13504/commits/d83988e064faf658882b2bae357ffe632c325958#diff-8141559b14bbf3947265d9b5fc691296R89 to extract our group name, or to add the group name somehow to the input list syntax.